addons/website/static/src/snippets/s_dynamic_snippet_carousel/000.js

```diff
@@ -26,19 +26,31 @@ const DynamicSnippetCarousel = DynamicSnippet.extend({
     //--------------------------------------------------------------------------
 
     /**
-     * Method to be overridden in child components in order to prepare QWeb
-     * options
-     * @private
+     * @override
+     */
+    _getQWebRenderOptions: function () {
+        return Object.assign(
+            this._super.apply(this, arguments),
+            {
+                interval: parseInt(this.$target[0].dataset.carouselInterval),
+            },
+        );
+    },
+    /**
+     * @deprecated
+     * @todo remove me in master, this was wrongly named and was supposed to
+     * override _getQWebRenderOptions. This is kept for potential custo in
```

Comment:
Hard-Nitpicking: Would have been cool to mention that it was introduced like that (and not the consequence of a later fix/imp) with https://github.com/odoo/odoo/commit/0e7640b5f22d2bea04bbe22d3189cff7e03af545

Reply:
It is okay, I still love you, I think